front-page.php

```diff
@@ -34,11 +29,11 @@
 
 		</main><!-- #main -->
 	</div><!-- #primary -->
-<?php get_template_part( 'template-parts/content', 'secondary' ) ?>	
-<?php get_sidebar(); ?>
+	<?php get_template_part( 'template-parts/front-page', 'secondary' );?>
```

Comment:
Do we need to maintain separate secondary content areas? Is this to switch out the flex content for the post listing?

Reply:
Yes. Content secondary shows flex content, which is used on other pages, but the home page displays the latest 2 blog posts.